.travis.yml

```diff
@@ -1,10 +1,11 @@
 language: python
-python: 3.5
-install: 
+python: 3.6
+cache: pip
+install:
   - ./setup
   - pip install coveralls
-script: 
+script:
   - coverage run rosie.py test
-
+  - python rosie.py run federal_senate
```

Comment:
Yes, that's the idea. In the line above we run all the written tests. Here, we `run federal_senate`. If doesn't work for any reason, will return a non-zero exit code and make the suite fail on CI.